src/components/utils/rhf-inputs/text-input.js

```diff
@@ -79,7 +82,7 @@ const TextInput = ({
             value={transformedValue}
             onChange={handleValueChanged}
             FormHelperTextProps={{
-                className: classes.helperText,
+                sx: helperTextStyle.helperText,
```

Comment:
Same as above

Reply:
I removed the useless FormHelperTextProps.